Patches/Vanilla Apparel Expanded/ThingDefs_Misc/Apparel_Medieval.xml

```diff
@@ -1,81 +1,100 @@
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_Cape == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_Cape"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<Bulk>7.5</Bulk>
-					<WornBulk>1.5</WornBulk>
-					<StuffEffectMultiplierArmor>1.8</StuffEffectMultiplierArmor>
-				</value>
-			</li>
+				<!-- == VAE_Headgear_TrapperHat == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Headgear_TrapperHat"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<Bulk>1.5</Bulk>
+						<StuffEffectMultiplierArmor>2.2</StuffEffectMultiplierArmor>
+					</value>
+				</li>
 
-			<!-- == VAE_Headgear_TrapperHat == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Headgear_TrapperHat"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<Bulk>1.5</Bulk>
-					<StuffEffectMultiplierArmor>2.2</StuffEffectMultiplierArmor>
-				</value>
-			</li>
+				<!-- == VAE_Headgear_SummerHat == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Headgear_SummerHat"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<StuffEffectMultiplierArmor>1.8</StuffEffectMultiplierArmor>
+					</value>
+				</li>
+				
+				<!-- == VAE_Headgear_SummerHat and VAE_Headgear_TrapperHat layers == -->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="VAE_Headgear_SummerHat" or defName="VAE_Headgear_TrapperHat"]/apparel/layers</xpath>
+					<value>
+						<li>MiddleHead</li>
+					</value>
+				</li>
 
-			<!-- == VAE_Headgear_SummerHat == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Headgear_SummerHat"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<StuffEffectMultiplierArmor>1.8</StuffEffectMultiplierArmor>
-				</value>
-			</li>
-			
-			<!-- == VAE_Headgear_SummerHat and VAE_Headgear_TrapperHat layers == -->
-			<li Class="PatchOperationAdd">
-				<xpath>Defs/ThingDef[defName="VAE_Headgear_SummerHat" or defName="VAE_Headgear_TrapperHat"]/apparel/layers</xpath>
-				<value>
-					<li>MiddleHead</li>
-				</value>
-			</li>
-		</operations>
+				<!-- == costStuffCounts == -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_Blouse"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>20</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_Tunic"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>25</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_Apron" or defName="VAE_Apparel_Cape"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>30</costStuffCount>
```

Comment:
The apron cost here seems quite high. I'd say 15 - 20, since it's not nearly a thick or insulating as a cape.